packages/components/src/components/dropdown/dropdown.js

```diff
@@ -304,14 +310,14 @@ class Dropdown extends mixin(
         if (text) {
           text.innerHTML = itemToSelect.innerHTML;
         }
-        itemToSelect.classList.add(this.options.classSelected);
+        itemToSelect.parentElement.classList.add(this.options.classSelected);
       }
       this.element.dataset.value = itemToSelect.parentElement.dataset.value;
 
       toArray(
         this.element.querySelectorAll(this.options.selectorItemSelected)
```

Comment:
Would we be able to use `selectorLinkSelected` here so we don't need to modify the `if()` logic in this loop?

Reply:
@asudoh I updated this section and it's ready for re-review